dvc/stage/__init__.py

```diff
@@ -472,7 +472,8 @@ def run(
             self.remove_outs(ignore_remove=False, force=False)
 
         if not self.frozen and self.is_import:
-            sync_import(self, dry, force)
+            jobs = kwargs.get("jobs", None)
```

Comment:
```suggestion
            jobs = kwargs.get("jobs")
```

Reply:
@skshetry . With this change tests on Windows would fail, I have reverted it.

Reply:
@karajan1001 Oops, didn't notice this before merging. I think windows tests were failing for an unrelated reason.  We've fixed them yesterday, they were failing because of gitpython.